crates/telio-wg/src/windows/service.rs

```diff
@@ -0,0 +1,93 @@
+#![cfg(windows)]
+
+use std::convert::{TryFrom, TryInto};
+use std::time::{Duration, Instant};
+use telio_utils::{telio_log_debug, telio_log_warn};
+use windows::{
+    core::{w, Error, Result as WindowsResult, HRESULT, PCWSTR},
+    Win32::{
+        Security::SC_HANDLE,
+        System::Services::{
+            OpenSCManagerW, OpenServiceW, QueryServiceStatus, SC_MANAGER_CONNECT,
+            SERVICE_QUERY_STATUS, SERVICE_STATUS,
+        },
+    },
+};
+
+pub const DEFAULT_SERVICE_WAIT_TIMEOUT: Duration = Duration::from_secs(5);
+
+#[derive(Debug)]
+enum WinServiceStatus {
+    Stopped = 1,
```

Comment:
would be nice to get a comment where the values are coming from :smile:

Reply:
They're taken from [here](https://learn.microsoft.com/en-us/windows/win32/api/winsvc/ns-winsvc-service_status). I did not find an enum in `windows-rs` crate, so made one from there.

Reply:
isn't this the one -> https://docs.rs/windows-service/latest/windows_service/service/enum.ServiceState.html

Reply:
ah but not in `windows-rs`

Reply:
Yeah the same, but that's a different crate, and I though siding with our plain implementation

Reply:
anyway, good enough :smile: